plugins/org.yakindu.sct.generator.c/src/org/yakindu/sct/generator/c/files/RxCHeader.xtend

```diff
@@ -27,23 +27,14 @@ class RxCHeader implements IContentTemplate {
 	@Inject extension GenmodelEntries
 
 	override content(ExecutionFlow it, GeneratorEntry entry, extension IGenArtifactConfigurations locations) '''
-		/*
-		 * sc_rxc.h
-		 *
-		 * Provided by YAKINDU Statechart Tools.
-		 *
-		 * Created on: 30.03.2020
-		 *     Author: terfloth
-		 */
 		«entry.licenseText»
 		
 		#ifndef «rxcModule.define»_H_
 		#define «rxcModule.define»_H_
 		
+		#include "stddef.h"
```

Comment:
please remove include - when we use sc_uinteger instead of size_t

Reply:
:+1: